source/addComment.js

```diff
@@ -1,3 +1,52 @@
 $(document).ready(function(){
-  // add your code here
+  $('#new_comment_button').click(function () {
```

Comment:
Love the organization of only having event listeners in doc ready and calling functions defined elsewhere!